apps/ubuntu_bootstrap/lib/pages/storage/passphrase/passphrase_model.dart

```diff
@@ -39,18 +39,32 @@ class PassphraseModel extends SafeChangeNotifier {
   bool get showPassphrase => _showPassphrase.value;
   set showPassphrase(bool value) => _showPassphrase.value = value;
 
+  /// Defines whether or not the user has selected TPM FDE
+  bool get isTpm => [
+        GuidedCapability.CORE_BOOT_ENCRYPTED,
+        GuidedCapability.CORE_BOOT_PREFER_ENCRYPTED,
+      ].contains(_service.guidedCapability);
+
   /// Whether the current input is valid.
   bool get isValid =>
       passphrase.isNotEmpty && passphrase == confirmedPassphrase;
 
+  /// Whether the user can skip providing a passphrase
+  bool get canSkip => isTpm ? passphrase.isEmpty : false;
```

Comment:
Nitpick: I think `isTpm && passphrase.isEmpty` is easier to understand

Reply:
Yeah that looks good.